decidim-admin/app/controllers/decidim/admin/features/base_controller.rb

```diff
@@ -8,7 +8,6 @@ module Features
       class BaseController < Admin::ApplicationController
         skip_authorize_resource
         include Concerns::ParticipatoryProcessAdmin
-        include NeedsParticipatoryProcess
```

Comment:
Mmmmm, no (although that's where I want to get at, yes). This is just an unnecessary include because the same functionality is already included via the `Concerns::ParticipatoryProcessAdmin` concern. See 0e894bc (my PR's are almost always best reviewed commit-per-commit :) ).